python/lsst/cp/pipe/cpCtiSolve.py

```diff
@@ -723,9 +735,17 @@ def findTraps(self, inputMeasurements, calib, detector):
             for exposureEntry in inputMeasurements:
                 exposureDict = exposureEntry['CTI']
                 if exposureDict[ampName]['IMAGE_MEAN'] < self.config.maxImageMean:
-                    signal.append(exposureDict[ampName]['IMAGE_MEAN'])
-                    data.append(exposureDict[ampName]['SERIAL_OVERSCAN_VALUES'][start:stop+1])
-                    new_signal.append(exposureDict[ampName]['LAST_COLUMN_MEAN'])
+                    imMean = exposureDict[ampName]['IMAGE_MEAN']
+                    sOverscan = exposureDict[ampName]['SERIAL_OVERSCAN_VALUES'][start: stop + 1]
+                    lastColumnMean = exposureDict[ampName]['LAST_COLUMN_MEAN']
+                    if not np.all(np.isfinite(imMean)) or not np.all(np.isfinite(sOverscan)) \
+                       or not np.all(np.isfinite(lastColumnMean)):
+                        self.log.warning("NaN detected in an entry for amplifier %s.", ampName)
+                        nSkipped += 1
+                    else:
+                        signal.append(imMean)
+                        data.append(sOverscan)
+                        new_signal.append(lastColumnMean)
                 else:
                     nSkipped += 1
             self.log.info("Skipped %d exposures brighter than %f.",
```

Comment:
Can we change this log message to note that we "Skipped %d exposures brighter than %f or non-finite"?  This counter is now not just "too bright."